troposphere.py

```diff
@@ -6,10 +6,13 @@ class Troposphere(Model):
     """
     Density as a function of altitude based on standard atmosphere model
 
-    Required pressures: upward pressure on rho and h
-    Assumptions: only valid to the top of the troposphere (~20km)
+    Required pressures: downward pressure on rho
+    Assumptions: only valid to the top of the troposphere (~11km)
```

Comment:
I think the "Required pressures" given here are much clearer than the `Assumptions` section of the current Sutherland docstring.